pynicotine/shares.py

```diff
@@ -838,16 +838,21 @@ def process_scanner_messages(self, scanner, scanner_queue):
         return False
 
     def check_shares_available(self):
+        return self.list_shares(only_unavailable=True)
+
+    def list_shares(self, only_unavailable=False):
```

Comment:
This method seems somewhat unnecessary. I'd simply use get_shared_folders() in the plugin command.

Reply:
Done